src/page-modules/assistant/details/trip-section/interchange-section.tsx

```diff
@@ -18,6 +18,7 @@ export type InterchangeSectionProps = {
   interchangeDetails: InterchangeDetails;
   publicCode?: string | null;
   maximumWaitTime?: number;
+  staySeated: boolean | undefined;
```

Comment:
Very much not happy with this where null | undefined | value etc. But not your fault and not specifically with this PR. But wouldn't there also be a case where this is null now?

Reply:
Is thath due to that both leg and leg.interchangeTo can be null? In that case, should also the null type be added to `maximumWaitTime` as `leg.interchangeTo.maximumWaitTime` can possibly also be null?

Reply:
It is due to serializing "over the wire" for server side communication in Next.js does not support undefined values, but marked as "null". so when mapping data it isn't undefined but null. In practise this is the same in most cases (null and undefined), but not for types.